src/app/main/api-gest-console/modules/aig-standard/components/cpv-new-update-dialog/cpv-new-update-dialog.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="dialog-content-wrapper">
+    <mat-toolbar matDialogTitle class="mat-accent m-0">
+        <mat-toolbar-row fxLayout="row" fxLayoutAlign="space-between center">
+            <span class="title dialog-title">Add cpv</span>
+            <button mat-icon-button (click)="matDialogRef.close()" aria-label="Close dialog">
+                <mat-icon>close</mat-icon>
+            </button>
+        </mat-toolbar-row>
+    </mat-toolbar>
+
+    <div mat-dialog-content class="p-24 pb-0 m-0" fusePerfectScrollbar>
+        <aig-cpv-new-update-form></aig-cpv-new-update-form>
```

Comment:
you need pass the current value if is a edt in this mode:
`<aig-cpv-new-update-form [cpv]="data.cpv"></aig-cpv-new-update-form>`